src/include/processor/operator/order_by/key_block_merger.h

```diff
@@ -27,6 +27,9 @@ struct StrKeyColInfo {
 };
 
 class MergedKeyBlocks {
+private:
```

Comment:
Should we move this to constant.h, since i saw orderbykeyencoder is always using/defining this constant.

Reply:
`BufferPoolConstants` is already a constant. I don't think we should put another constant in common. This should be local to the class.